packages/apm/src/span.ts

```diff
@@ -191,13 +192,20 @@ export class Span implements SpanInterface, SpanContext {
     return span;
   }
 
+  /**
+   * @inheritDoc
+   */
+  public isChildSpan(): boolean {
+    return this._parentSpanId !== undefined;
+  }
+
   /**
    * Continues a trace from a string (usually the header).
    * @param traceparent Traceparent string
    */
   public static fromTraceparent(
     traceparent: string,
-    spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'spanId' | 'sampled' | 'traceid'>>,
+    spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'spanId' | 'sampled' | 'traceid' | 'parentSpanId'>>,
```

Comment:
```suggestion
    spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'spanId' | 'sampled' | 'traceId' | 'parentSpanId'>>,
```

It's a shame the compiler doesn't complain about keys that are not in the referenced type :cry: